src/main/java/com/minecolonies/entity/ai/citizen/fisherman/Pond.java

```diff
@@ -54,8 +56,10 @@ private static boolean checkWater(IBlockAccess world, BlockPos water)
         int z = water.getZ();
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Local variable 'z' could be declared final [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=pmd%3ALocalVariableCouldBeFinal)